Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -5,71 +5,28 @@
 using System.Threading.Tasks;
 using Doppler.HtmlEditorApi.DopplerSecurity;
 using Microsoft.AspNetCore.Authorization;
-using Microsoft.Extensions.Logging;
 using Doppler.HtmlEditorApi.Model;
+using Doppler.HtmlEditorApi.Infrastructure;
 
 namespace Doppler.HtmlEditorApi.Controllers
 {
     [Authorize]
     [ApiController]
     public class CampaignsController
     {
+        private readonly IDummyRepository _dummyRepository;
```

Comment:
The interface is an abstraction, the client should not know if it refers to a dummy implementation or not.

```suggestion
        private readonly IRepository _repository;
```

Reply:
Ok, I thought the real one could be independent.

Reply:
Fixed